src/core/settings.c

```diff
@@ -373,12 +379,16 @@ QuicSettingApply(
         Destination->IsSet.RetryMemoryLimit = TRUE;
     }
     if (Source->IsSet.LoadBalancingMode && (!Destination->IsSet.LoadBalancingMode || OverWrite)) {
-        if (Source->LoadBalancingMode > QUIC_LOAD_BALANCING_SERVER_ID_IP) {
+        if (Source->LoadBalancingMode > QUIC_LOAD_BALANCING_SERVER_ID_FIXED) {
```

Comment:
Better to define a MAX value in QUIC_LOAD_BALANCING_MODE, so that this bit of code isn't dependent on the order of the modes in the enum

Reply:
Will do.